README.md

```diff
@@ -143,6 +144,23 @@ brew install nginx-full --with-tarantool-module
 
 [Back to contents](#contents)
 
+## Test run and notes for contributors
+
+Firstly, run all tests with `./test/auto.sh`. Before test running
+itself it creates necessary environment for testing.
+
+For selective running, start:
+  1) nginx in other terminal with `./nginx/objs/nginx -c conf/nginx.conf`,

```

Comment:
That does not work for me due to several reasons:

1. There is no `conf/nginx.conf` file in the repository root.
2. If `test/ngx_confs/nginx.dev.conf` was assumed, we cannot just add `-c test/ngx_confs/nginx.dev.conf`. The config is searched in a system directory, but even if we'll pass it by a real path, nginx will try to find `mime.types` in a wrong location and will try to write logs into a system directory.
3. All those problems appears, because I tried to follow the readme and assume that the "Build via nginx 'configure'" section is the prerequisite for running the tests. This, however, is not so: here we assume that nginx is built using `make configure-for-testing && make -j` [^1] from this repository root (and `nginx` source repository is placed in the root of this repository). and so `./nginx/objs/nginx` should be called without `-c`. Please, clarify how to build nginx + module in order to run tests: it seems the process differs from building for production use.

Please, test your instructions on a clean repository (without any gitignored files, without nginx sources — from scratch).

[^1]: It is built with custom prefix directory this way (`test-root/`) and nginx configuration for testing is copied inside this directory.

Reply:
I've clarified `README.md`. I tested it on clean repository just following README guide. It works fine. I've never used `make configure-for-testing`.

Reply:
> I've never used `make configure-for-testing`.

And `test/auto.sh`, which calls it?

Reply:
No, I used it. Аnyway, I double-checked. Everything works.

Reply:
I re-read the intruction and, okay, I found where was my mistake. The 'Build from source' instruction suggests `make build-all`, which works for me (together with `make configure-for-testing` afterwards). When I did it the first time, I wrongly followed the "Build via nginx 'configure'" instruction.